homework/01LessonOne.js

```diff
@@ -202,12 +207,14 @@ var stringFourStrictNotEqualsNumberFour;
 //
 // What’s the difference between == and ===?
 //
-// Your Answer Goes Here:
+// Your Answer Goes Here: so == means something is equal to something and === means
+// something is equal to the value of something else
```

Comment:
Good start. [Strict Equality](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness) is the key phrase here.